bokehjs/src/lib/models/tools/math_text.ts

```diff
@@ -0,0 +1,217 @@
+import * as p from "core/properties"
+import * as visuals from "core/visuals"
+import {isNumber} from "core/util/types"
+import {Context2d} from "core/util/canvas"
+import {ImageLoader} from "core/util/image"
+import {CanvasImage} from "models/glyphs/image_url"
+import {Model} from "../../model"
+import {color2css} from "core/util/color"
+import {Size} from "core/types"
+import {View} from "core/view"
+
+declare global {
+  interface Window {
+    _bokeh_mathjax_instantiated: boolean
+  }
+}
+
+type Position = {
+  sx: number
+  sy: number
+  x_anchor?: number | "left" | "center" | "right"
+  y_anchor?: number | "top"  | "center" | "baseline" | "bottom"
```

Comment:
Can you actually get the baseline out of what MathJax produces? I.e. if we get a collection of rendered labels, will we be able to align them along their baselines? Additionally, math text may contain other axes of alignment (e.g. alignment of fractions along the rule separating numerator from denominator); can we access those?

Reply:
I don't think we can, I'm treating it as a fixed imaged, we can control the width and height and it's font will change to better accommodate to the new dimensions but we can not control the inner elements

Reply:
We should open an issue or support discussion with MathJax to describe our situation and ask for guidance. MathJax is an NF project, I can also try to make direct introductions with the main maintainers if need be.